sfm is disabled because subdirectory libmv not found

Windows 10 VS 2015 win 64 opencv opencv_contrib branch even with master

My bash script with  cmake command : 
`
myRepo=$(pwd)
CMAKE_OPTIONS='-DBUILD_PERF_TESTS:BOOL=OFF -DBUILD_TESTS:BOOL=OFF -DBUILD_DOCS:BOOL=OFF -DWITH_CUDA:BOOL=OFF -DBUILD_EXAMPLES:BOOL=ON  -DWITH_LAPACK:BOOL=ON -DINSTALL_CREATE_DISTRIB=ON'

cmake -G"Visual Studio 14 2015 Win64"  \
-DEIGEN_DIR:PATH="$myRepo"/eigen -DEIGEN_INCLUDE_DIR="$myRepo"/eigen -DEIGEN_INCLUDE_PATH="$myRepo"/eigen \
-DCeres_DIR="$myRepo"/install/ceres-solver/cmake -Dglog_DIR:PATH="$myRepo"/install/glog/lib/cmake/glog -Dgflags_DIR:PATH="$myRepo"/build/gflags \
$CMAKE_OPTIONS -DOPENCV_EXTRA_MODULES_PATH=../../opencv_contrib/modules \
-DCMAKE_INSTALL_PREFIX=../../install/"$RepoSource" -DVTK_DIR="$myRepo"/install/vtk/lib/cmake/vtk-8.1  ../../"$RepoSource"
`

my cmake output is  : 

-- The CXX compiler identification is MSVC 19.0.24215.1
-- The C compiler identification is MSVC 19.0.24215.1
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/x86_amd64/cl.exe
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/x86_amd64/cl.exe -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/x86_amd64/cl.exe
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/x86_amd64/cl.exe -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Performing Test HAVE_CXX11 (check file: cmake/checks/cxx11.cpp)
-- Performing Test HAVE_CXX11 - Success
-- Found PythonInterp: F:/Python/Python35/python.exe (found suitable version "3.5", minimum required is "2.7")
-- Found PythonInterp: F:/Python/Python35/python.exe (found suitable version "3.5", minimum required is "3.4")
-- Found PythonLibs: optimized;F:/Python/Python35/libs/python35.lib;debug;F:/Python/Python35/libs/python35_d.lib (found suitable exact version "3.5.0")
-- Performing Test HAVE_CXX_FSIGNED_CHAR
-- Performing Test HAVE_CXX_FSIGNED_CHAR - Failed
-- Performing Test HAVE_C_FSIGNED_CHAR
-- Performing Test HAVE_C_FSIGNED_CHAR - Failed
-- Performing Test HAVE_CPU_SSE3_SUPPORT (check file: cmake/checks/cpu_sse3.cpp)
-- Performing Test HAVE_CPU_SSE3_SUPPORT - Success
-- Performing Test HAVE_CPU_SSSE3_SUPPORT (check file: cmake/checks/cpu_ssse3.cpp)
-- Performing Test HAVE_CPU_SSSE3_SUPPORT - Success
-- Performing Test HAVE_CPU_SSE4_1_SUPPORT (check file: cmake/checks/cpu_sse41.cpp)
-- Performing Test HAVE_CPU_SSE4_1_SUPPORT - Success
-- Performing Test HAVE_CPU_POPCNT_SUPPORT (check file: cmake/checks/cpu_popcnt.cpp)
-- Performing Test HAVE_CPU_POPCNT_SUPPORT - Success
-- Performing Test HAVE_CPU_SSE4_2_SUPPORT (check file: cmake/checks/cpu_sse42.cpp)
-- Performing Test HAVE_CPU_SSE4_2_SUPPORT - Success
-- Performing Test HAVE_CPU_FP16_SUPPORT (check file: cmake/checks/cpu_fp16.cpp)
-- Performing Test HAVE_CPU_FP16_SUPPORT - Success
-- Performing Test HAVE_CXX_ARCH:AVX (check file: cmake/checks/cpu_avx.cpp)
-- Performing Test HAVE_CXX_ARCH:AVX - Success
-- Performing Test HAVE_CXX_ARCH:AVX2 (check file: cmake/checks/cpu_avx2.cpp)
-- Performing Test HAVE_CXX_ARCH:AVX2 - Success
-- Performing Test HAVE_CPU_BASELINE_FLAGS
-- Performing Test HAVE_CPU_BASELINE_FLAGS - Success
-- Performing Test HAVE_CPU_DISPATCH_FLAGS_SSE4_1
-- Performing Test HAVE_CPU_DISPATCH_FLAGS_SSE4_1 - Success
-- Performing Test HAVE_CPU_DISPATCH_FLAGS_SSE4_2
-- Performing Test HAVE_CPU_DISPATCH_FLAGS_SSE4_2 - Success
-- Performing Test HAVE_CPU_DISPATCH_FLAGS_FP16
-- Performing Test HAVE_CPU_DISPATCH_FLAGS_FP16 - Success
-- Performing Test HAVE_CPU_DISPATCH_FLAGS_AVX
-- Performing Test HAVE_CPU_DISPATCH_FLAGS_AVX - Success
-- Performing Test HAVE_CPU_DISPATCH_FLAGS_AVX2
-- Performing Test HAVE_CPU_DISPATCH_FLAGS_AVX2 - Success
-- Check if the system is big endian
-- Searching 16 bit integer
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Using unsigned short
-- Check if the system is big endian - little endian
-- Looking for fseeko
-- Looking for fseeko - not found
-- Check size of off64_t
-- Check size of off64_t - failed
-- Looking for assert.h
-- Looking for assert.h - found
-- Looking for fcntl.h
-- Looking for fcntl.h - found
-- Looking for io.h
-- Looking for io.h - found
-- Looking for jbg_newlen
-- Looking for jbg_newlen - not found
-- Looking for search.h
-- Looking for search.h - found
-- Looking for string.h
-- Looking for string.h - found
-- FFMPEG: Download: opencv_ffmpeg.dll
-- FFMPEG: Download: opencv_ffmpeg_64.dll
-- FFMPEG: Download: ffmpeg_version.cmake
-- IPPICV: Download: ippicv_2017u3_win_intel64_general_20170822.zip
-- found Intel IPP (ICV version): 2017.0.3 [2017.0.3]
-- at: G:/Lib/build/opencv/3rdparty/ippicv/ippicv_win
-- found Intel IPP IW sources: 2017.0.3
-- at: G:/Lib/build/opencv/3rdparty/ippicv/ippicv_win/../ippiw_win/
-- Could not find OpenBLAS include. Turning OpenBLAS_FOUND off
-- Could not find OpenBLAS lib. Turning OpenBLAS_FOUND off
-- Looking for pthread.h
-- Looking for pthread.h - not found
-- Found Threads: TRUE
-- A library with BLAS API not found. Please specify library location.
-- LAPACK requires BLAS
-- A library with LAPACK API not found. Please specify library location.
-- Found JNI: C:/Program Files/Java/jdk1.8.0_112/lib/jawt.lib
-- Could NOT find Pylint (missing: PYLINT_EXECUTABLE)
-- Could NOT find Matlab (missing: MATLAB_MEX_SCRIPT MATLAB_INCLUDE_DIRS MATLAB_ROOT_DIR MATLAB_LIBRARIES MATLAB_LIBRARY_DIRS MATLAB_MEXEXT MATLAB_ARCH MATLAB_BIN)
-- Found VTK ver. 8.1.0 (usefile: G:/Lib/Install/vtk/lib/cmake/vtk-8.1/UseVTK.cmake)
-- Caffe:   NO
-- Protobuf:   NO
-- Glog:   NO
-- tiny-dnn: Download: v1.0.0a3.tar.gz
-- Looking for tiny_dnn.h
-- Looking for tiny_dnn.h - found
-- Found tiny-dnn in: G:/Lib/build/opencv/3rdparty/tinydnn/tiny-dnn-1.0.0a3
-- Performing Test COMPILER_SUPPORTS_CXX11
-- Performing Test COMPILER_SUPPORTS_CXX11 - Success
-- Module opencv_dnn_modern disabled because Protobuf is not found
-- freetype2:   NO
-- harfbuzz:    NO
-- No preference for use of exported gflags CMake configuration set, and no hints for include/library directories provided. Defaulting to preferring an installed/exported gflags CMake configuration if available.
-- Found installed version of gflags: G:/lib/build/gflags
-- Detected gflags version: 2.2.1
-- Found required Ceres dependency: Eigen version 3.3.90 in G:/Lib/eigen
-- Found installed version of glog: G:/lib/Install/glog/lib/cmake/glog
-- Detected glog version: 0.3.5
-- Found required Ceres dependency: glog
-- Found Ceres version: 1.13.0 installed in: G:/Lib/install/ceres-solver with components: [SchurSpecializations, OpenMP]
-- Checking SFM deps... TRUE
-- Processing WORLD modules...
--     module opencv_core...
--     module opencv_flann...
--     module opencv_imgproc...
--     module opencv_ml...
--     module opencv_objdetect...
--     module opencv_phase_unwrapping...
--     module opencv_photo...
--     module opencv_plot...
--     module opencv_reg...
--     module opencv_surface_matching...
--     module opencv_video...
--     module opencv_viz...
--     module opencv_xphoto...
--     module opencv_bgsegm...
--     module opencv_dnn...
-- Torch importer has been enabled. To run the tests you have to install Torch ('th' executable should be available) and generate testdata using opencv_extra/testdata/dnn/generate_torch_models.py script.
--     module opencv_face...
--     module opencv_fuzzy...
--     module opencv_imgcodecs...
--     module opencv_shape...
--     module opencv_videoio...
--     module opencv_xobjdetect...
--     module opencv_highgui...
--     module opencv_superres...
--     module opencv_bioinspired...
--     module opencv_dpm...
--     module opencv_features2d...
--     module opencv_line_descriptor...
--     module opencv_saliency...
--     module opencv_text...
-- Tesseract:   NO
--     module opencv_calib3d...
--     module opencv_ccalib...
--     module opencv_datasets...
--     module opencv_rgbd...
--     module opencv_stereo...
--     module opencv_structured_light...
--     module opencv_tracking...
--     module opencv_videostab...
--     module opencv_xfeatures2d...
-- xfeatures2d/boostdesc: Download: boostdesc_bgm.i
-- xfeatures2d/boostdesc: Download: boostdesc_bgm_bi.i
-- xfeatures2d/boostdesc: Download: boostdesc_bgm_hd.i
-- xfeatures2d/boostdesc: Download: boostdesc_binboost_064.i
-- xfeatures2d/boostdesc: Download: boostdesc_binboost_128.i
-- xfeatures2d/boostdesc: Download: boostdesc_binboost_256.i
-- xfeatures2d/boostdesc: Download: boostdesc_lbgm.i
-- xfeatures2d/vgg: Download: vgg_generated_48.i
-- xfeatures2d/vgg: Download: vgg_generated_64.i
-- xfeatures2d/vgg: Download: vgg_generated_80.i
-- xfeatures2d/vgg: Download: vgg_generated_120.i
--     module opencv_ximgproc...
--     module opencv_aruco...
--     module opencv_optflow...
--     module opencv_sfm...
-- No preference for use of exported gflags CMake configuration set, and no hints for include/library directories provided. Defaulting to preferring an installed/exported gflags CMake configuration if available.
-- Found installed version of gflags: G:/lib/build/gflags
-- Detected gflags version: 2.2.1
-- Found required Ceres dependency: Eigen version 3.3.90 in G:/Lib/eigen
-- Found installed version of glog: G:/lib/Install/glog/lib/cmake/glog
-- Detected glog version: 0.3.5
-- Found required Ceres dependency: glog
-- Found Ceres version: 1.13.0 installed in: G:/Lib/install/ceres-solver with components: [SchurSpecializations, OpenMP]
-- Checking SFM deps... TRUE
CMake Error at G:/Lib/opencv_contrib/modules/sfm/CMakeLists.txt:116 (add_subdirectory):
  add_subdirectory given source "src/libmv_light" which is not an existing
  directory.
Call Stack (most recent call first):
  modules/world/CMakeLists.txt:13 (include)
  modules/world/CMakeLists.txt:32 (include_one_module)


--     module opencv_stitching...
-- Processing WORLD modules... DONE
-- Excluding from source files list: G:/Lib/build/opencv/modules/world/accum.neon.cpp
-- Processing module opencv_img_hash...
-- DNN_FACE_DETECTOR: Download: res10_300x300_ssd_iter_140000.caffemodel
--
-- General configuration for OpenCV 3.3.0-dev =====================================
--   Version control:               3.2.0-1401-g1114cbf67
--
--   Extra modules:
--     Location (extra):            G:/Lib/opencv_contrib/modules
--     Version control (extra):     3.2.0-381-g24b8f251
--
--   Platform:
--     Timestamp:                   2017-08-31T22:05:55Z
--     Host:                        Windows 10.0.15063 AMD64
--     CMake:                       3.9.1
--     CMake generator:             Visual Studio 14 2015 Win64
--     CMake build tool:            C:/Program Files (x86)/MSBuild/14.0/bin/MSBuild.exe
--     MSVC:                        1900
--
--   CPU/HW features:
--     Baseline:                    SSE SSE2 SSE3
--       requested:                 SSE3
--     Dispatched code generation:  SSE4_1 SSE4_2 FP16 AVX AVX2
--       requested:                 SSE4_1 SSE4_2 AVX FP16 AVX2
--       SSE4_1 (2 files):          + SSSE3 SSE4_1
--       SSE4_2 (1 files):          + SSSE3 SSE4_1 POPCNT SSE4_2
--       FP16 (1 files):            + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 AVX
--       AVX (5 files):             + SSSE3 SSE4_1 POPCNT SSE4_2 AVX
--       AVX2 (7 files):            + SSSE3 SSE4_1 POPCNT SSE4_2 FP16 FMA3 AVX AVX2
--
--   C/C++:
--     Built as dynamic libs?:      YES
--     C++11:                       YES
--     C++ Compiler:                C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/x86_amd64/cl.exe  (ver 19.0.24215.1)
--     C++ flags (Release):         /DWIN32 /D_WINDOWS /W4 /GR  /EHa  /D _CRT_SECURE_NO_DEPRECATE /D _CRT_NONSTDC_NO_DEPRECATE /D _SCL_SECURE_NO_WARNINGS /Gy /bigobj /Oi      /wd4251 /wd4324 /wd4275 /wd4512 /wd4589 /MP12  /MD /O2 /Ob2 /DNDEBUG  /Zi
--     C++ flags (Debug):           /DWIN32 /D_WINDOWS /W4 /GR  /EHa  /D _CRT_SECURE_NO_DEPRECATE /D _CRT_NONSTDC_NO_DEPRECATE /D _SCL_SECURE_NO_WARNINGS /Gy /bigobj /Oi      /wd4251 /wd4324 /wd4275 /wd4512 /wd4589 /MP12  /MDd /Zi /Ob0 /Od /RTC1
--     C Compiler:                  C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin/x86_amd64/cl.exe
--     C flags (Release):           /DWIN32 /D_WINDOWS /W3  /D _CRT_SECURE_NO_DEPRECATE /D _CRT_NONSTDC_NO_DEPRECATE /D _SCL_SECURE_NO_WARNINGS /Gy /bigobj /Oi        /MP12  /MD /O2 /Ob2 /DNDEBUG  /Zi
--     C flags (Debug):             /DWIN32 /D_WINDOWS /W3  /D _CRT_SECURE_NO_DEPRECATE /D _CRT_NONSTDC_NO_DEPRECATE /D _SCL_SECURE_NO_WARNINGS /Gy /bigobj /Oi        /MP12  /MDd /Zi /Ob0 /Od /RTC1
--     Linker flags (Release):      /machine:x64  /INCREMENTAL:NO  /debug
--     Linker flags (Debug):        /machine:x64  /debug /INCREMENTAL
--     ccache:                      NO
--     Precompiled headers:         NO
--     Extra dependencies:
--     3rdparty dependencies:
--
--   OpenCV modules:
--     To be built:                 core flann imgproc ml objdetect phase_unwrapping photo plot reg surface_matching video viz xphoto bgsegm dnn face fuzzy imgcodecs shape videoio xobjdetect highgui superres bioinspired dpm features2d line_descriptor saliency text calib3d ccalib datasets rgbd stereo structured_light tracking videostab xfeatures2d ximgproc aruco optflow sfm stitching world img_hash
--     Disabled:                    contrib_world
--     Disabled by dependency:      -
--     Unavailable:                 cudaarithm cudabgsegm cudacodec cudafeatures2d cudafilters cudaimgproc cudalegacy cudaobjdetect cudaoptflow cudastereo cudawarping cudev java python2 python3 ts cnn_3dobj cvv dnn_modern freetype hdf matlab
--
--   Windows RT support:            NO
--
--   GUI:
--     QT:                          NO
--     Win32 UI:                    YES
--     OpenGL support:              NO
--     VTK support:                 YES (ver 8.1.0)
--
--   Media I/O:
--     ZLib:                        build (ver 1.2.8)
--     JPEG:                        build (ver 90)
--     WEBP:                        build (ver encoder: 0x020e)
--     PNG:                         build (ver 1.6.24)
--     TIFF:                        build (ver 42 - 4.0.2)
--     JPEG 2000:                   build (ver 1.900.1)
--     OpenEXR:                     build (ver 1.7.1)
--     GDAL:                        NO
--     GDCM:                        NO
--
--   Video I/O:
--     Video for Windows:           YES
--     DC1394 1.x:                  NO
--     DC1394 2.x:                  NO
--     FFMPEG:                      YES (prebuilt binaries)
--       avcodec:                   YES (ver 57.89.100)
--       avformat:                  YES (ver 57.71.100)
--       avutil:                    YES (ver 55.58.100)
--       swscale:                   YES (ver 4.6.100)
--       avresample:                YES (ver 3.5.0)
--     GStreamer:                   NO
--     OpenNI:                      NO
--     OpenNI PrimeSensor Modules:  NO
--     OpenNI2:                     NO
--     PvAPI:                       NO
--     GigEVisionSDK:               NO
--     DirectShow:                  YES
--     Media Foundation:            NO
--     XIMEA:                       NO
--     Intel PerC:                  NO
--
--   Parallel framework:            Concurrency
--
--   Trace:                         YES (with Intel ITT)
--
--   Other third-party libraries:
--     Use Intel IPP:               2017.0.3 [2017.0.3]
--                at:               G:/Lib/build/opencv/3rdparty/ippicv/ippicv_win
--     Use Intel IPP IW:            sources (2017.0.3)
--                   at:            G:/Lib/build/opencv/3rdparty/ippicv/ippicv_win/../ippiw_win/
--     Use Lapack:                  NO
--     Use Eigen:                   YES (ver 3.3.90)
--     Use Cuda:                    NO
--     Use OpenCL:                  YES
--     Use OpenVX:                  NO
--     Use custom HAL:              NO
--
--   OpenCL:                        <Dynamic loading of OpenCL library>
--     Include path:                G:/Lib/opencv/3rdparty/include/opencl/1.2
--     Use AMDFFT:                  NO
--     Use AMDBLAS:                 NO
--
--   Python 2:
--     Interpreter:                 NO
--
--   Python 3:
--     Interpreter:                 F:/Python/Python35/python.exe (ver 3.5)
--
--   Python (for build):            F:/Python/Python35/python.exe
--
--   Java:
--     ant:                         NO
--     JNI:                         C:/Program Files/Java/jdk1.8.0_112/include C:/Program Files/Java/jdk1.8.0_112/include/win32 C:/Program Files/Java/jdk1.8.0_112/include
--     Java wrappers:               NO
--     Java tests:                  NO
--
--   Matlab:                        Matlab not found or implicitly disabled
--
--   Tests and samples:
--     Tests:                       NO
--     Performance tests:           NO
--     C/C++ Examples:              YES
--
--   Install path:                  G:/Lib/install/opencv
--
--   cvconfig.h is in:              G:/Lib/build/opencv
-- -----------------------------------------------------------------
--
-- Configuring incomplete, errors occurred!
See also "G:/Lib/build/opencv/CMakeFiles/CMakeOutput.log".
See also "G:/Lib/build/opencv/CMakeFiles/CMakeError.log".


sfm is found is twice (why?) and second time  message is 
`
CMake Error at G:/Lib/opencv_contrib/modules/sfm/CMakeLists.txt:116 (add_subdirectory):
  add_subdirectory given source "src/libmv_light" which is not an existing
  directory.
`
How to solve this problem?

@KenjiKyo Thanks for PR.
@alalek 
```
It does not solve problem in my configuration : 
--     module opencv_ximgproc...
--     module opencv_aruco...
--     module opencv_optflow...
--     module opencv_sfm...
-- No preference for use of exported gflags CMake configuration set, and no hints for include/library directories provided. Defaulting to preferring an installed/exported gflags CMake configuration if available.
-- Found installed version of gflags: G:/lib/build/gflags
-- Detected gflags version: 2.2.1
-- Found required Ceres dependency: Eigen version 3.3.90 in G:/Lib/eigen
-- Found installed version of glog: G:/lib/Install/glog/lib/cmake/glog
-- Detected glog version: 0.3.5
-- Found required Ceres dependency: glog
-- Found Ceres version: 1.13.0 installed in: G:/Lib/install/ceres-solver with components: [SchurSpecializations, OpenMP]
-- Checking SFM deps... TRUE
CMake Error at G:/Lib/opencv_contrib/modules/sfm/CMakeLists.txt:116 (add_subdirectory):
  add_subdirectory not given a binary directory but the given source
  directory "G:/Lib/opencv_contrib/modules/sfm/src/libmv_light" is not a
  subdirectory of "G:/Lib/opencv/modules/world".  When specifying an
  out-of-tree source a binary directory must be explicitly specified.
Call Stack (most recent call first):
  modules/world/CMakeLists.txt:13 (include)
  modules/world/CMakeLists.txt:32 (include_one_module)


--     module opencv_stitching...
-- Processing WORLD modules... DONE
```
Using "Visual Studio 15 2017 Win64" gives me same results.
What is your cmake configuration ? you must have -DINSTALL_CREATE_DISTRIB=ON in cmake

--- a/modules/sfm/CMakeLists.txt
+++ b/modules/sfm/CMakeLists.txt
@@ -1,3 +1,4 @@
+set(OPENCV_MODULE_IS_PART_OF_WORLD FALSE)
 set(the_description "SFM algorithms")
 
 
@@ -113,7 +114,7 @@ if(NOT CMAKE_VERSION VERSION_LESS 2.8.11) # See ocv_target_include_directories()
   endif()
 endif()
 include_directories(${OCV_TARGET_INCLUDE_DIRS_${the_module}})
-add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/src/libmv_light)
+add_subdirectory("${CMAKE_CURRENT_LIST_DIR}/src/libmv_light" "${CMAKE_CURRENT_BINARY_DIR}/src/libmv")
 
 ocv_target_link_libraries(${the_module} ${LIBMV_LIGHT_LIBS})
 

--- a/modules/sfm/CMakeLists.txt
+++ b/modules/sfm/CMakeLists.txt
@@ -113,7 +113,7 @@ if(NOT CMAKE_VERSION VERSION_LESS 2.8.11) # See ocv_target_include_directories()
   endif()
 endif()
 include_directories(${OCV_TARGET_INCLUDE_DIRS_${the_module}})
-add_subdirectory(src/libmv_light)
+add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/src/libmv_light)
 
 ocv_target_link_libraries(${the_module} ${LIBMV_LIGHT_LIBS})
 

--- a/modules/sfm/CMakeLists.txt
+++ b/modules/sfm/CMakeLists.txt
@@ -39,7 +39,7 @@ endif()
 ### LIBMV LIGHT DEFINITIONS ###
 
 set(LIBMV_LIGHT_INCLUDES
-  src/libmv_light
+  "${CMAKE_CURRENT_LIST_DIR}/src/libmv_light"
   "${OpenCV_SOURCE_DIR}/include/opencv"
   "${GLOG_INCLUDE_DIRS}"
   "${GFLAGS_INCLUDE_DIRS}"

